firmware/components/micropython/lib/utils/pyexec.c

```diff
@@ -501,6 +501,7 @@ int pyexec_friendly_repl(void) {
             }
         }
 
+        mp_hal_stdout_tx_str("\r\n");
```

Comment:
This fixes the python console not doing a newline when you press enter, but instead only doing a carriage return